File: Rasp/01/led.py

import os
import time
import RPi.GPIO as GPIO

LED_PIN = 21  # Change to GPIO21 (Pin 40)

GPIO.setmode(GPIO.BCM)
GPIO.setup(LED_PIN, GPIO.OUT)

# Open VS Code
os.system("code &")

try:
    while True:
        GPIO.output(LED_PIN, GPIO.HIGH)  # LED ON
        time.sleep(1)
        GPIO.output(LED_PIN, GPIO.LOW)   # LED OFF
        time.sleep(1)
except KeyboardInterrupt:
    GPIO.cleanup()

File: Rasp/01/led.c
#include <wiringPi.h>
#include <stdio.h>
#include <unistd.h>

#define LED_PIN 21  // GPIO21 (Physical Pin 40)

int main() {
    if (wiringPiSetupGpio() == -1) {
        printf("Setup Failed!\n");
        return 1;
    }

    pinMode(LED_PIN, OUTPUT);

    while (1) {
        digitalWrite(LED_PIN, HIGH);
        sleep(1);
        digitalWrite(LED_PIN, LOW);
        sleep(1);
    }

    return 0;
}
